ci-operator/step-registry/upi/conf/vsphere/ova/windows/upi-conf-vsphere-ova-windows-commands.sh

```diff
@@ -11,7 +11,7 @@ echo "$(date -u --rfc-3339=seconds) - Configuring govc exports."
 source "${SHARED_DIR}/govc.sh"
 
 # TODO: read VM template information from environment variable
-VM_TEMPLATE="windows-golden-images/windows-server-2004-template"
+VM_TEMPLATE="windows-golden-images/windows-server-2022-template-platform-none"
```

Comment:
We will need to get this new image replicated to the IBM cloud environment, if not already done.

Reply:
Correct

Reply:
Yes I am aware of this. I was hoping to do some dev e2e testing but that's looking tough at the moment, so I will go ahead and ask them to copy the image after I rename it.